internal/backend/auth/authhttpmiddleware/authmiddleware.go

```diff
@@ -83,6 +83,11 @@ func newSessionsMiddleware(next http.Handler, sessions authsessions.Store) http.
 				return
 			}
 
+			if session == nil {
+				http.Error(w, "invalid session", http.StatusUnauthorized)
+				return
+			}
+
 			if session != nil {
```

Comment:
Due to the `if` above, you don't need this one.

Reply:
removed this section from code